### PR TITLE
Hemibrain example for nat.examples

### PR DESCRIPTION
**Pull request by alexanderbates**
+ Find MBONs
+ Get MBONs
+ Plot MBONs
+ MBON connectivity
+ Paths onto PPL1-a'3
+ Get MB meshes

### CODE REVIEW

**Comment by alexanderbates** [comment]
@jefferis permission to pull?

File: 15-hemibrain/00-setup.R [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# install
+if(!require('devtools')) install.packages("devtools")
+if(!require('natverse')) devtools::install_github("natverse/natverse")
+if(!require('ComplexHeatmap')) install.packages("ComplexHeatmap")
+if(!require('devtools')) install.packages("ggnetwork")
```
Comment (jefferis) [inline 15-hemibrain/00-setup.R]: ggnetwork?

File: 15-hemibrain/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Central brain drosophila neurons from the FlyEM HemiBrain project
+
+The largest draft connectome to date was released on the 22nd of January, 2020. The [FlyEM Janelia HemiBrain project](https://www.janelia.org/project-team/flyem/hemibrain) has released 21,662, many more fragments, and almost 10 million synapses. About 35% of its uatomatically predicted synapses have been matched up to a semi-automatically traced 'body' deemed large enough to constitute and identifiable neuron. This data can be accessed using our R package [neuprintr](https://github.com/natverse/neuprintr/) to query a publically accessible [neuPrint explorer instance](https://neuprint.janelia.org/).
```
Comment (jefferis) [inline 15-hemibrain/README.md]: 21,662 neurons

Comment (jefferis) [reply 15-hemibrain/README.md]: uatomatically

Comment (jefferis) [reply 15-hemibrain/README.md]: and identifiable neuron

**Pull request merged**